File: Bab 8/8.6.2.3 Stair Word.java
import java.util.Scanner;

public class Main{

	public static void main(String[] args) {
		
		String[] string = new String[10];
		int baris = 0, kolom = 0, spasi = 0;

		Scanner sc = new Scanner(System.in);

		for (baris = 0; baris < 9; baris++) {

			System.out.println("Masukan angka:");

			try{

				string[baris] = sc.next();

			}catch(Exception e){

			}
		}

		for (baris = 0; baris < 9; baris++)
		{
			for (kolom = 0; kolom < spasi; kolom++)
			{
				System.out.print(" ");
			}

			System.out.println(string[baris]);

			spasi = spasi + string[baris].length();
		}
	}
}

File: Bab 8/8.6.2.3 Stair Word.c
#include <stdio.h>
#include <string.h>

int main()
{
	
	char string[9][50];
	int baris, kolom, spasi;

	for (baris = 0; baris < 9; baris++)
	{
		
		printf("Masukan kata:\n");
		scanf("%s", &string[baris]);
		printf("\n");

	}

	spasi = 0;

	for (baris = 0; baris < 9; baris++)
	{
		for (kolom = 0; kolom < spasi; kolom++)
		{
			printf(" ");
		}

		printf("%s\n", string[baris]);

		spasi = spasi + strlen(string[baris]);
	}

	return 0;
}